test/tests/functional/pbs_soft_walltime.py

```diff
@@ -162,6 +170,20 @@ def test_soft_walltime_perms(self):
         self.server.expect(JOB, 'Resource_List.soft_walltime',
                            op=UNSET, id=jid)
 
+        operator = str(OPER_USER) + '@*'
+        self.server.manager(MGR_CMD_SET, SERVER,
+                            {'operators': (INCR, operator)},
+                            sudo=True)
+
+        try:
```

Comment:
So, it seems like both normal users AND operators cannot alter soft walltime. If that's the case, then I'd modify the docstring of this test to also specify that the test checks that operators can't alter it either, along with normal users.

Reply:
ok..